### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by merlimat**

### CODE REVIEW

File: site/_includes/navbar.html [hunk 1]
```diff
@@ -1,60 +1,75 @@
-<nav class="navbar navbar-toggleable-md navbar-light container-fluid sticky-top">
+<nav class="navbar navbar-toggleable-md navbar-light container-fluid sticky-top"
```
Comment (lucperkins) [inline site/_includes/navbar.html]: I'd prefer to keep CSS out of HTML elements but I can do that myself later (my current Sass setup needs some streamlining and simplification)

File: site/_layouts/default.html [hunk 2]
```diff
@@ -33,5 +33,16 @@
 
     <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
     {% endif %}
+
+<script>
```
Comment (lucperkins) [inline site/_layouts/default.html]: Google Analytics should be used only when the site is running non-locally, in "production." I know a good way to ensure this, which I can apply on my own.

File: site/_includes/navbar.html (hunk 1 above)
Comment (merlimat) [inline site/_includes/navbar.html]: Yes, this was a quick&dirty. I'll fix it.

Comment (merlimat) [inline site/_includes/navbar.html]: Fixed here: https://github.com/streamlio/incubator-pulsar/commit/8897abac8b8cb2ff56e5bdadac32df467212f67e

**Comment by merlimat** [comment]
Closing this PR. I have pushed these changes in the `docs` branch, on top of your last changes

**Pull request closed without merging**